.github/workflows/release.yml

```diff
@@ -3,24 +3,16 @@
 # SPDX-License-Identifier: Apache-2.0
 
 name: Release
+
 on:
-  workflow_dispatch:
-    inputs:
-      release:
-        description: 'Fabric CA Release, e.g. 1.5.6'
-        required: true
-        type: string
-      two_digit_release:
-        description: 'Fabric CA Two Digit Release, e.g. 1.5'
-        required: true
-        type: string
-      commit_hash:
-        description: 'Commit hash, e.g. df9c661a192f8cf11376d9d643a0021f1a76c34b'
-        required: true
-        type: string
+  push:
+    tags: [ v* ]
```

Comment:
We intentionally picked a workflow_dispatch manual action for a more controlled 'all-in-one' release process, rather than the automagic trigger in the background based on a tag, but let's see what others think.

Reply:
This actually _is_ supported well by the "all in one" release process. 

With the existing practices, the release process involves manually launching a workflow action / dispatch event.  (There is a step in the workflow that creates the release in GH through a custom action.)

With the "GitHub Way," the order is reversed.  The _human action_ is to create the release (through GUI, etc.), and in response GH will apply a tag to your target branch, which then triggers the release action.

Practically speaking, both models can be supported easily.  The key here in this PR is that the revision metadata is inferred by the TAG triggering the event.  If we want to keep the existing dispatch trigger that's OK - it should just be a single text box entry for the driver to input the semrev value for the release being cranked.

While the order is a little weird, and the "magic tag" seems .... magical ....   it is actually less error prone than manually entering the target for the release, IMO.   I'm OK with either - provided that the input to the process (either way) is a *semrev* string.

Also there are really good mechanics that come from using the implicit branch tags and names.   This took a few hours to grok, but there is some pure gold in the way [docker/metadata-action](https://github.com/docker/metadata-action#typeref) is inferring coordinates from tags and git relative positions.  Good stuff.

BTW the native arm64 performance is insanely fast, compared to QEMU emulation.   Happy to flex the release trigger to however we want it to run.

Hope to see this running on a Raspberry Pi, soon.  (Too bad they are made from _unobtanium!_)